meadow-client/src/Types.purs

```diff
@@ -90,16 +90,24 @@ instance showView :: Show View where
   show = gShow
 
 type FrontendState
-  = {view :: View, runResult :: RemoteData AjaxError (Either InterpreterError RunResult), marloweCompileResult :: Either (Array MarloweError) Unit, authStatus :: RemoteData AjaxError AuthStatus, createGistResult :: RemoteData AjaxError Gist, marloweState :: MarloweState, oldContract :: Maybe String}
+  = { view :: View
+    , compilationResult :: RemoteData AjaxError (Either InterpreterError (InterpreterResult RunResult))
+    , warnings :: Array Warning
```

Comment:
Hmm....something's wrong there. Surely the warnings only exist if `compilationResult` is `Success _`?

Reply:
that property should have been deleted.